doc/api-documentation.rst

```diff
@@ -53,4 +53,11 @@ API Documentation
 .. automodule:: oled.virtual
     :members:
     :undoc-members:
-    :show-inheritance:
\ No newline at end of file
+    :show-inheritance:
+
+:mod:`oled.error`
```

Comment:
@thijstriemstra Can you reorder so that these sections are sorted alphabetically ?